src/EventListener/PrettyErrorScreenListener.php

```diff
@@ -68,12 +74,18 @@ class PrettyErrorScreenListener
      * @param bool              $prettyErrorScreens True to render the error screens
      * @param \Twig_Environment $twig               The twig environment
      * @param ConfigAdapter     $config             The config adapter
+     * @param LoggerInterface   $logger             The logger service
      */
-    public function __construct($prettyErrorScreens, \Twig_Environment $twig, ConfigAdapter $config)
-    {
+    public function __construct(
+        $prettyErrorScreens,
+        \Twig_Environment $twig,
+        ConfigAdapter $config,
+        LoggerInterface $logger
```

Comment:
Does this really make sense? We are only using the logger if we render a pretty error screen and the pretty error screen says "check the log file for the actual error message". Would anyone want to render the pretty error screen without logging the actual error message?